daily_word_bot/utils.py

```diff
@@ -102,10 +102,18 @@ def build_available_commands_msg(bot_commands: List[BotCommand]) -> str:
 hero_char = "🦸🏿‍♀️"
 
 
+def convert_to_hyperlink(url: str, text: str) -> str:
+    return f"<a href=\"{url}\">{text}</a>"
+
+
+def convert_to_github_hyperlink(contributor_name: str) -> str:
+    return convert_to_hyperlink(f"https://github.com/{contributor_name}", contributor_name)
+
+
 def build_info_msg(version: str, start_date: datetime,
                    word_bank_length: int, word_bank_update_date: datetime,
                    contributors: List[Dict]) -> str:
-    str_contributors = "\n".join([f" {hero_char} {c['login']}" for c in contributors if c['type'] != 'Bot'])
+    str_contributors = "\n".join([f" {hero_char} {utils.convert_to_github_hyperlink(c['login'])}" for c in contributors if c['type'] != 'Bot'])
```

Comment:
use directly the hyperlink function, the utl can be found in c["html_url"], since it is already provided in the json response ->https://api.github.com/repos/rooom13/dailyWord-bot/contributors